src/agent/config-example.py

```diff
@@ -0,0 +1,10 @@
+#INFLUX
+INFLUXDB_ORGANIZATION = "agh-utc"
+INFLUXDB_URL = "http://localhost:8086"
+INFLUXDB_BUCKET = "dockeria"
+INFLUXDB_TOKEN = "yourtokenhere"
+INFLUXDB_WRITE_INTERVAL_TIME = 10
```

Comment:
Considering fact that Influx is now optional and accessed only from and by backend component, do we still need those variables, or at least named like that?

Reply:
Sure thang